app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Turniquette</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
```

Comment:
* Turbolinks... useless since we're creating an API.

* I would maintain CSRF
Just read some Strack Ov. threads and blog posts with explicits "don't do that" about disabling csrf protection for APIs.
As a suggestion about how to act: http://jamescpoole.com/2013/10/31/rails-4-csrf-protection-with-clients-using-apis/